files/default/neutron-ha-tool.py

```diff
@@ -305,14 +304,15 @@ def run(args):
     return 1 if errors > 0 else 0
 
 
-def l3_agent_rebalance(qclient, noop=False, wait_for_router=True):
+def l3_agent_rebalance(qclient, args):
```

Comment:
Agreed with @aplanas here, better explicit than implicit. As far as I can see, we are not saving a lot of args to counter the simplicity loss of the functions usability :/